lib/std/Build.zig

```diff
@@ -408,6 +408,22 @@ fn createChildOnly(parent: *Build, dep_name: []const u8, build_root: Cache.Direc
     return child;
 }
 
+/// Returns true if T is *const [_:0]u8
+fn isStringLiteral(comptime T: type) bool {
+    switch (@typeInfo(T)) {
+        .Pointer => |ptr_info| {
+            if (T != *const ptr_info.child) return false;
+            switch (@typeInfo(ptr_info.child)) {
+                .Array => |arr_info| {
+                    if (T != *const [arr_info.len:0]u8) return false;
```

Comment:
This could be made more direct and remove a negation by changing it to:
```
   return (T == *const [arr_info.len:0]u8);
```

Reply:
Yeah, that's a result of not paying attention while rewriting. Thanks!